src/index.html

```diff
@@ -13,6 +13,42 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <div class="product-card__info">
+        <img
+          class="product-card__image"
+          src="images/imac.jpeg"
```

Comment:
The 'alt' attribute should be more descriptive of the image content for accessibility reasons. Consider using a brief description of the actual product.